### DISCUSSION TITLE
When I deploy minio in 4Node cluster mode, the WebUI crashed.

### DISCUSSION THREAD

**Discussion by Juminiy** [minio/minio#20196] [Q&A] [answered]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Expected Behavior
When I open the Firefox, Bing, Chrome, in windows, linux, macOS they occurs to be in console: 

Uncaught SyntaxError: Unexpected token '<' (at main.b9463185.js:1:1)

## Steps to Reproduce (for bugs)
```bash
# !/bin/bash
node_num=$1
gatewy_ip=192.168.31.1
node_1_ip=192.168.31.66
node_2_ip=192.168.31.67
node_3_ip=192.168.31.68
node_4_ip=192.168.31.69

ETC_HOSTS=/etc/hosts
TIME_SYNCD_CONF=/etc/systemd/timesyncd.conf
ETC_MINIO=/etc/default/minio
ETC_FSTAB=/etc/fstab
MINIO_DEB=minio_20240715190230.0.0_amd64.deb


if [ "$node_num" = "" ]; then
    echo node_num can not be null
    exit 1
fi


echo checking network conn
ping -c 3 $gatewy_ip
ping -c 3 $node_1_ip
ping -c 3 $node_2_ip
ping -c 3 $node_3_ip
ping -c 3 $node_4_ip


echo setting minio hosts
sudo hostnamectl hostname minio$node_num.cluster
echo '# minio hosts BEGIN    ' >> $ETC_HOSTS
echo $node_1_ip minio1.cluster >> $ETC_HOSTS
echo $node_2_ip minio2.cluster >> $ETC_HOSTS
echo $node_3_ip minio3.cluster >> $ETC_HOSTS
echo $node_4_ip minio4.cluster >> $ETC_HOSTS
echo '# minio hosts END      ' >> $ETC_HOSTS


#echo setting time sync 
#echo FallbackNTP=ntp.aliyun.com >> $TIME_SYNCD_CONF
#sudo systemctl restart systemd-timesyncd


echo setting file/disk
sudo mkfs.xfs -f /dev/sdb
sudo mkdir -p /data
echo -n "/dev/sdb "         >> $ETC_FSTAB
echo -n "/data "            >> $ETC_FSTAB
echo -n "xfs "              >> $ETC_FSTAB
echo -n "defaults,noatime " >> $ETC_FSTAB
echo -n "0 "                >> $ETC_FSTAB
echo -n "2"                 >> $ETC_FSTAB
sudo mount -av


echo installing minio
echo modify perm of minio.deb
sudo chmod 777 $MINIO_DEB
sudo dpkg -i $MINIO_DEB
sudo groupadd -r minio-user
sudo useradd -M -r -g minio-user minio-user
sudo chown -R minio-user:minio-user /data

sudo rm -f $ETC_MINIO
echo MINIO_VOLUMES="http://minio{1...4}.cluster:9000/data/minio"     >> $ETC_MINIO
echo MINIO_OPTS="--console-address :9001"                            >> $ETC_MINIO
echo MINIO_ROOT_USER=minioadmin                                      >> $ETC_MINIO
echo MINIO_ROOT_PASSWORD=Minio@123456                                >> $ETC_MINIO
echo MINIO_BROWSER_REDIRECT_URL="http://minio.cluster:8080/minio/ui" >> $ETC_MINIO

sudo systemctl restart minio.service
sudo systemctl enable minio.service
```
download .deb pkg
and run `sudo sh install_minio.sh 1` in node1, ... node2, node3, node4 the service show that 

● minio.service - MinIO
     Loaded: loaded (/lib/systemd/system/minio.service; enabled; vendor preset: enabled)
     Active: active (running) since Wed 2024-07-31 07:23:47 UTC; 35min ago
       Docs: https://docs.min.io
   Main PID: 1507 (minio)
      Tasks: 9
        CPU: 5.704s
     CGroup: /system.slice/minio.service
             └─1507 /usr/local/bin/minio server --console-address :9001 http://minio{1...4}.cluster:9000/data/minio

Jul 31 07:23:47 minio3.cluster minio[1507]: ---------------------------
Jul 31 07:23:47 minio3.cluster minio[1507]: MinIO Object Storage Server
Jul 31 07:23:47 minio3.cluster minio[1507]: Copyright: 2015-2024 MinIO, Inc.
Jul 31 07:23:47 minio3.cluster minio[1507]: License: GNU AGPLv3 - https://www.gnu.org/licenses/agpl-3.0.html
Jul 31 07:23:47 minio3.cluster minio[1507]: Version: RELEASE.2024-07-15T19-02-30Z (go1.22.5 linux/amd64)
Jul 31 07:23:47 minio3.cluster minio[1507]: API: http://192.168.31.69:9000  http://127.0.0.1:9000
Jul 31 07:23:47 minio3.cluster minio[1507]: WebUI: http://minio.cluster:8080/minio/ui
Jul 31 07:23:47 minio3.cluster minio[1507]: Docs: https://min.io/docs/minio/linux/index.html
Jul 31 07:23:47 minio3.cluster minio[1507]: ---------------------------
Jul 31 07:23:48 minio3.cluster minio[1507]: INFO: IAM load(startup) finished. (duration: 14.201686ms)

everything backend is ok, but the frontend WebUI Crashed!!!!!! I can bear!!!

## Your Environment
* Version used (`minio --version`): 
minio version RELEASE.2024-07-15T19-02-30Z (commit-id=ded373e6008e1a746875c9c0d15e7e9911ff1549)
Runtime: go1.22.5 linux/amd64
License: GNU AGPLv3 - https://www.gnu.org/licenses/agpl-3.0.html
Copyright: 2015-2024 MinIO, Inc.

* Server setup and configuration: above 

* Operating System and version (`uname -a`):
* Linux minio1.cluster 5.15.0-117-generic #127-Ubuntu SMP Fri Jul 5 20:13:28 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
* Linux minio2.cluster 5.15.0-117-generic #127-Ubuntu SMP Fri Jul 5 20:13:28 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
* Linux minio3.cluster 5.15.0-117-generic #127-Ubuntu SMP Fri Jul 5 20:13:28 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
* Linux minio4.cluster 5.15.0-117-generic #127-Ubuntu SMP Fri Jul 5 20:13:28 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

**Comment by harshavardhana** [comment, marked as answer]
> ```shell
> echo MINIO_BROWSER_REDIRECT_URL="http://minio.cluster:8080/minio/ui" >> $ETC_MINIO
> ```

Is this resolvable from your browser?

**Comment by Juminiy** [comment]
solved
